src/twisted/mail/pop3.py

```diff
@@ -565,7 +565,9 @@ def state_COMMAND(self, line):
             return self.processCommand(*line.split(' '))
         except (ValueError, AttributeError, POP3Error, TypeError) as e:
             log.err()
-            self.failResponse('bad protocol or server: %s: %s' % (e.__class__.__name__, e))
+            self.failResponse(
+                        'bad protocol or server: %s: %s' % (e.__class__.__name__, e)
```

Comment:
I am trying to understand what are the rules for this indentation... I see this double indentation in other parts of this PR.